src/usr/pnor/sfc_memboot_ast2400.H

```diff
@@ -0,0 +1,138 @@
+/* IBM_PROLOG_BEGIN_TAG                                                   */
+/* This is an automatically generated prolog.                             */
+/*                                                                        */
+/* $Source: src/usr/pnor/sfc_memboot_ast2400.H $                          */
+/*                                                                        */
+/* OpenPOWER HostBoot Project                                             */
+/*                                                                        */
+/* Contributors Listed Below - COPYRIGHT 2014                             */
+/* [+] Google Inc.                                                        */
+/* [+] International Business Machines Corp.                              */
+/*                                                                        */
+/*                                                                        */
+/* Licensed under the Apache License, Version 2.0 (the "License");        */
+/* you may not use this file except in compliance with the License.       */
+/* You may obtain a copy of the License at                                */
+/*                                                                        */
+/*     http://www.apache.org/licenses/LICENSE-2.0                         */
+/*                                                                        */
+/* Unless required by applicable law or agreed to in writing, software    */
+/* distributed under the License is distributed on an "AS IS" BASIS,      */
+/* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or        */
+/* implied. See the License for the specific language governing           */
+/* permissions and limitations under the License.                         */
+/*                                                                        */
+/* IBM_PROLOG_END_TAG                                                     */
+#ifndef __PNOR_SFCMEMBOOTAST2400_H
+#define __PNOR_SFCMEMBOOTAST2400_H
+
+#include <limits.h>
+#include <targeting/common/targetservice.H>
+#include <errl/errlentry.H>
+#include "sfcdd.H"
+#include <config.h>
+
+/** @file sfc_ast2400.H
+ *  @brief Provides the logic to access and configure the
+ *         AST2400 BMC in order to access the PNOR
+ */
+
+/**
+ *  @brief  AST2400 SFC Device Driver Class
+ *    Provides the logic to access and configure the
+ *    AST2400 BMC in order to access the PNOR
+ */
+class SfcMembootAST2400 : public SfcDD
```

Comment:
Can we just inherit from SfcAST2400?  Seems like a lot of boilerplate in here that is probably already in that file as well.